foxpuppet/browser/base_window.py

```diff
@@ -2,12 +2,8 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from foxpuppet.foxpuppet import FoxPuppet
 
+class BaseWindow(object):
```

Comment:
Right now it's not. I think it was going to be the browserwindow, before we changed the hierarchy.